src/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <div class="brand">
+          <a href="/" class="logo"><i class="fa-solid fa-utensils"></i> </a>
+          <h1 class="site-title">TheRecipeApp</h1>
+        </div>
+        <ul class="nav-links main-menu">
+          <li class="nav-item">
+            <p class="nav-link">Categories:</p>
+          </li>
+          <li class="nav-item">
+            <a id="beef" href="#" class="nav-link"
+              >Beef <span class="meal-count"></span
+            ></a>
+          </li>
+          <li class="nav-item">
+            <a id="seafood" href="#" class="nav-link"
+              >Seafood <span class="meal-count"></span
+            ></a>
+          </li>
+          <li class="nav-item">
+            <a id="dessert" href="#" class="nav-link"
+              >Dessert <span class="meal-count"></span
+            ></a>
+          </li>
+        </ul>
+      </div>
+      <div class="meal-wrapper">
+        <div class="meal-search">
+          <h2 class="title">Find Meals with Ingredient</h2>
+          <blockquote>
+            Real food doesn't have ingredients, real food is ingredients
+            <br />
+            <cite>Jamie oliver</cite>
+          </blockquote>
+          <div class="meal-search-box">
+            <input
+              type="text"
+              class="search-control"
+              placeholder="Enter an ingredient"
+              id="search-input"
+            />
+            <button type="submit" class="search-btn btn" id="search-btn">
+              <i class="fas fa-search"></i>
+            </button>
+          </div>
+        </div>
```

Comment:
- [OPTIONAL] The search functionality is a  very good feature but didn't work on my side. I will suggest you take a look at it to improve your project.